CHANGES.rst

```diff
@@ -8,9 +8,14 @@ New Tools and Services
 API changes
 -----------
 
+query.py
```

Comment:
please move this into `Infrastructure, Utility and Other Changes and Additions` and no need to add `query.py` heading.